### PR TITLE
fix NUMPY INCLUDE with Cython not installed

### PR DESCRIPTION
**Pull request by Baaaaam**
fixes some issues with ubuntu 14.04.

I am not sure yet if it actually add `NUMPY_INCLUDE_DIR` to the `INCLUDE_DIRECTORIES` or overnight them....

### CODE REVIEW

**Comment by Baaaaam** [comment]
the PR #1320 also includes this fix...

File: CMakeLists.txt [hunk 1]
```diff
@@ -307,9 +305,10 @@ IF(NOT CYCLUS_DOC_ONLY)
         "${SQLITE3_INCLUDE_DIR}"
         "${HDF5_INCLUDE_DIRS}"
         "${Boost_INCLUDE_DIR}"
-        "${COIN_INCLUDE_DIRS}"
-        "${NUMPY_INCLUDE_DIRS}")
-
+        "${COIN_INCLUDE_DIRS}")
+    if(Cython_FOUND)
+      INCLUDE_DIRECTORIES("${NUMPY_INCLUDE_DIRS}")
+  endif(Cython_FOUND)
```
Comment (scopatz) [inline CMakeLists.txt]: Consistent indentation

**Comment by scopatz** [comment]
Thanks @Baaaaam, this seems like a reasonable change with the one suggestion.

**Comment by Baaaaam** [comment]
closing this one since #1320 include this change

**Pull request closed without merging**